### ISSUE TITLE
Success Criterion Email 2.4.3 (Focus Order) - Example Text is Cutoff

### ISSUE THREAD

**Issue by Grunet** [Grunet/digestible-wcag#4]
Compare what's shown in the email ([link to in-browser version](https://htmlpreview.github.io/?https://raw.githubusercontent.com/Grunet/digestible-wcag-sc-emails/master/dist/2-4-3.html))

![image](https://user-images.githubusercontent.com/29322882/96376552-3d12f400-1145-11eb-932d-be3e0546d3ae.png)

To what's on the [WCAG understanding page](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html#examples)

![image](https://user-images.githubusercontent.com/29322882/96376582-7a778180-1145-11eb-83ce-0727356efa3e.png)

**Comment by Grunet** [comment]
This is coming from the examples not having been scraped correctly in this case. 

Notice how there are 5 examples in total for this success criterion on [the understanding page](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html#examples) but only 3 appearing in [the scraped JSON](https://raw.githubusercontent.com/Grunet/digestible-wcag-scraping/master/dist/wcag22.json), all coming from the 4th and 5th examples' text

![image](https://user-images.githubusercontent.com/29322882/96376643-d3dfb080-1145-11eb-8a43-ed8bc005f09a.png)

![image](https://user-images.githubusercontent.com/29322882/96376668-eeb22500-1145-11eb-9fc5-c1aa10a53c9d.png)

**Comment by Grunet** [comment]
Specifically, [the scraping code](https://github.com/Grunet/digestible-wcag-scraping/blob/master/src/understandingPages/scrape.ts) is only looking for the paragraph content within the examples section in this case, rather than the list content ("content without paragraphs") that is actually holding each example.

Here's what the markup of [the examples section](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html#examples) looks like. Notice that there are 5 list elements corresponding to the 5 examples, but that the 4th and 5th list elements also contain paragraph elements (which is what's currently confusing the scraping code into only trying to pull the paragraphs' content)
![image](https://user-images.githubusercontent.com/29322882/96376854-f920ee80-1146-11eb-93d8-00d8fdac0524.png)

Here's the code that's scraping the paragraph info
![image](https://user-images.githubusercontent.com/29322882/96376777-8879d200-1146-11eb-8e26-94e5ebd1f876.png)


Beneath it is the code that it should be running instead, to scrape the list info (but isn't because the previous code has already found some paragraphs in the content)
![image](https://user-images.githubusercontent.com/29322882/96376796-a7786400-1146-11eb-9274-ba77347a15e4.png)

**Comment by Grunet** [comment]
Switching to having the code search for list elements first seems like it might work to fix this (though I'm not sure how that would affect other cases), otherwise a more nuanced change might be needed

Also it's unclear how many other success criteria are affected by this, but I'll try to keep this updated as they come along.

**Comment by Grunet** [comment]
Incidentally addressed this while updating everything to the May 2020 version of WCAG 2.2, where I ended up deciding to get rid of any "Note" section in the examples to handle an issue 3.2.7 was having (matching how the main success criteria text is handled too).

In this case everything is the same, there's just no longer a dangling "Note" text at the bottom
